manifests/ops-files/set-fs-inotify-limit.yml

```diff
@@ -0,0 +1,19 @@
+- type: replace
+  path: /addons/-
```

Comment:
Because this is applied through addons, it's going to be added to all instance groups indiscriminately, including the `windows_workers` group running on a Windows OS. `os-conf` obviously won't work outside of Linux-based VMs. We should break this down and explicitly add the job to `master` and `worker` explicitly (as annoying as the duplication might be).

Additionally, is there any analog that we might need to add for Windows? I know it has similar notification functionality but I'm not familiar enough to know whether it has similar limitations.

Reply:
You can use [placement rules](https://bosh.io/docs/runtime-config/#placement-rules) to control which stemcell (aka OS) the addons end up on.

Reply:
We have discussed with the Windows PKS team and will work with them on it in the future. Currently, we do not support our workloads running on Windows workers.

That being said are the windows workers different from the linux workers? Asking because if we add a placement rule for jobs `master` and `worker`, will that include windows workers?

The other thing is, in the future we are hoping to run an agent that collects logs/metrics from all vms in a PKS cluster. More of an "Alana" perspective...so with that reasoning we want this to be configured on all vms. Maybe we can add an exclude rule for windows vms for now?

Reply:
@tvs @benmoss I've updated our addon to only run on machines with ubuntu stemcells. I think this will work for us in the near future.

Reply:
Excluding `windows_workers` can work too.

Reply:
Windows workers are under an instance group called [windows-worker](https://github.com/cloudfoundry-incubator/kubo-deployment/blob/be9aa11ae1a57a2dfdd1e35af6aa7558792205f4/manifests/ops-files/windows/add-worker.yml#L19) so they wouldn't be affected by the addon for `worker`.

Reply:
@benmoss yeah...I saw and that and just thought the stemcell placement rule would work just as well for our purposes. Thanks.